test/helpers/lobbyHelper.js

```diff
@@ -0,0 +1,37 @@
+export default async function openSession(participant) {
+    if (participant.moderator) {
```

Comment:
I'm confused, how can a participant have a moderator property without joining?

Reply:
However, that has changed with a new approach